biojava-structure/src/main/java/org/biojava/nbio/structure/PDBStatus.java

```diff
@@ -18,134 +18,66 @@
- * PDB website.</a>
- *
- * <p>PDB supersessions form a directed acyclic graph, where edges point from an
- * obsolete ID to the entry that directly superseded it. For example, here are
- * edges from one portion of the graph:<br/>
- *
- * 1CAT -> 3CAT<br/>
- * 3CAT -> 7CAT<br/>
- * 3CAT -> 8CAT<br/>
+ * <p>
+ * All methods query the
+ * <a href="https://data.rcsb.org">
+ * RCSB Data REST API</a>
+ * <p>
  *
- * <p>The methods {@link #getReplaces(String, boolean) getReplaces(pdbId, false)}/
- * {@link #getReplacement(String, boolean, boolean) getReplacement(pdbId, false, true)}
- * just get the incoming/outgoing edges for a single node. The recursive versions
- * ({@link #getReplaces(String, boolean) getReplaces(pdbId, true)},
- * {@link #getReplacement(String, boolean, boolean) getReplacement(pdbId, true, false)})
- * will do a depth-first search up/down the tree and return a list of all nodes ]
- * reached.
- *
- * <p>Finally, the getCurrent() method returns a single PDB ID from among the
- * results of
- * {@link #getReplacement(String, boolean) getReplacement(pdbId, true)}.
- * To be consistent with the old REST ordering, this is the PDB ID that occurs
- * last alphabetically.
- *
- * <p>Results are cached to reduce server load.
- *
- * @author Spencer Bliven <sbliven@ucsd.edu>
+ * @author Spencer Bliven
  * @author Amr AL-Hossary
+ * @author Jose Duarte
  * @since 3.0.2
  */
 public class PDBStatus {
 
 	private static final Logger logger = LoggerFactory.getLogger(PDBStatus.class);
 
-	public static final String DEFAULT_PDB_SERVER = "www.rcsb.org";
-	public static final String PDB_SERVER_PROPERTY = "PDB.SERVER";
-
-	/**
-	 * saves the returned results for further use.
-	 *
-	 */
-	//TODO Use SoftReferences to allow garbage collection
-	private static Map<String, Map<String, String>> recordsCache= new Hashtable<String, Map<String, String>>();
+	public static final String DEFAULT_RCSB_DATA_API_SERVER = "data.rcsb.org";
+	public static final String ALL_CURRENT_ENDPOINT = "https://%s/rest/v1/holdings/current/entry_ids";
+	public static final String STATUS_ENDPOINT = "https://%s/rest/v1/holdings/status/%s";
+	public static final String STATUS_LIST_ENDPOINT = "https://%s/rest/v1/holdings/status?ids=%s";
 
 	/**
-	 * Represents the status of PDB IDs. 'OBSOLETE' and 'CURRENT' are the most
-	 * common.
-	 * @author Spencer Bliven <sbliven@ucsd.edu>
-	 *
+	 * Represents a simplified 3 state status of PDB IDs.
+	 * @author Spencer Bliven
 	 */
 	public enum Status {
-		OBSOLETE,
+		// the simplified status enum in rcsb_repository_holdings_combined
+		REMOVED,
 		CURRENT,
-		AUTH,
-		HOLD,
-		HPUB,
-		POLC,
-		PROC,
-		REFI,
-		REPL,
-		WAIT,
-		WDRN,
-		MODEL,
-		UNKNOWN;
-
+		UNRELEASED;
 
 		/**
-		 *
-		 * @param statusStr
-		 * @return
 		 * @throws IllegalArgumentException If the string is not recognized
 		 */
 		public static Status fromString(String statusStr) {
 			Status status;
 			String statusStrUpper = statusStr.toUpperCase();
```

Comment:
Nitpicking here: I'd remove the `toUpperCase` call (as `equalsIgnoreCase` is used anyway) and change the order to `if("REMOVED".equalsIgnoreCase(statusStr))`. That way `statusStr` can be `null` which is nice to have for a method with public visibility.